src/printf.cpp

```diff
@@ -242,8 +318,8 @@ void process_printf(char*& data, const printf_descriptor_map_t& descs,
 
     auto output = printf_out.str();
     if (printf_cb != nullptr) {
-        auto len = output.size();
-        printf_cb(output.c_str(), len, data >= data_end, printf_userdata);
+        printf_cb(output.c_str(), output.size(), data >= data_end,
+                  printf_userdata);
```

Comment:
What does it fix?

Reply:
nothing I just though it was more compact. Ill put it back to the way it was.